client/src/components/Views/SeeMoreDrawer.js

```diff
@@ -26,22 +26,22 @@ export function SeeMoreDrawer() {
   const seeMoreCardContent = useMemo(() => {
     return tripData.map((item, index) => {
       const dayNumber = index + 1
-      const colorName = getColorName(item['color'])
+      const colorName = getTailwindName(item[0]['colorNumber'])
```

Comment:
whats the item[0] here and would item['colorNumber'] work here?

Reply:
This assumes when stages are fetched, an array of stages, i.e.
```
[ {stageNumber: 1, colorNumber: 21},  {stageNumber: 2, colorNumber: 34} ]
```

If we do `item['colorNumber']` then we would need to fetch an object from the backend. But since the colorNumber is only stored in the stages and we completely don't have the concept of a day, I think we might need to include the `[0]`?

Reply:
Could we rename `item` to `stage` and could we rename `tripData` to be `stages`? :')

Reply:
ah good point, i've renamed `item` to `day`. `tripData` is actually the data of the trip so i'll leave that as is